### ISSUE TITLE
--show-cops does not show custom cops

### ISSUE THREAD

**Issue by scgruber** [bbatsov/rubocop#3020]
Adding a custom cop should cause the cop to display in `rubocop --show-cops`, but it doesn't.

I think this is because is `act_on_options` passes through to `handle_exiting_options` and prints the cops **before** it loads the config, which means it has an incomplete list of cops.

```ruby
    def act_on_options
      handle_exiting_options

      ConfigLoader.debug = @options[:debug]
      ConfigLoader.auto_gen_config = @options[:auto_gen_config]

      @config_store.options_config = @options[:config] if @options[:config]

      if @options[:color]
        # color output explicitly forced on
        Rainbow.enabled = true
      elsif @options[:color] == false
        # color output explicitly forced off
        Rainbow.enabled = false
      end
    end

    def handle_exiting_options
      return unless Options::EXITING_OPTIONS.any? { |o| @options.key? o }

      puts RuboCop::Version.version(false) if @options[:version]
      puts RuboCop::Version.version(true) if @options[:verbose_version]
      print_available_cops if @options[:show_cops]
      raise Finished
    end
```

## Expected behavior

`$ rubocop --show-cops` should display custom cops that are included by the `require` option in `.rubocop.yml`.

## Actual behavior

Custom cops included by a `require` in `.rubocop.yml` are not shown in the list of cops.

## Steps to reproduce the problem

1. Write a custom cop
1. Require the cop in `.rubocop.yml`
1. Run `rubocop --show-cops`

## RuboCop version

```
$ bundle exec rubocop -V
0.37.2 (using Parser 2.3.0.6, running on ruby 1.9.3 i486-linux)
```

**Comment by bbatsov** [comment]
This was recently fixed on `master`.
